pyrasterframes/src/main/python/setup.py

```diff
@@ -53,6 +53,7 @@ class PweaveDocs(distutils.cmd.Command):
     user_options = [
         # The format is (long option, short option, description).
         ('files=', 'f', 'Specific files to pweave. Defaults to all in `docs` directory.'),
+        ('doctype=', 'd', 'Output format type. Defaults to `markdown`')
```

Comment:
so confusingly `-f` is the doctype option shortcut on the `pweave` cli

so maybe the files option is here `-s` for sourcefile and `-f` is doctype. That would help bring those in line for the future and break only a little code

Reply:
Heh. I'll fix. I'd pulled those out of the air.